Makefile

```diff
@@ -198,17 +209,17 @@ db-refresh: ## Stop and delete the database container and volume, then rebuilds
 elastic-update: ## Run the elasticsearch migration (n=migration name, r=rollback, i=ignore errors).
 	$(info Run the elasticsearch migration (n=$(n)))
 	@./db/elasticsearch/scripts/migration.sh $(if $(n),-n $(n),"") $(if $(r),-r,"") $(if $(i),-i,"")
-#  $(if $(r),-r,"")
-# ifdef ERROR1
-# 	$(error failed to run migration)
-# endif
 
 ##############################################################################
-# Kafka Utility Commands
+# Kafka Commands
 ##############################################################################
 
 reset-consumer-offset: ## Reset the consumer group topic offset.
 	$(info Reset the consumer group topic offset)
 	@cd ./db/kafka/broker/scripts; ./reset-consumer-offset.sh
 
+kafka-update: ## Run the kafka migration (n=migration name, r=rollback, z=zookeeper)
```

Comment:
We can now run scripts based on a version number for kafka initialization and migrations